frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -63,6 +65,10 @@ const OnboardingWrapper = ({ children }: { children: React.ReactNode }): JSX.Ele
             const OtherProductsStep = <OnboardingOtherProductsStep stepKey={OnboardingStepKey.OTHER_PRODUCTS} />
             steps = [...steps, OtherProductsStep]
         }
+        if (featureFlags[FEATURE_FLAGS.INVITE_TEAM_MEMBER_ONBOARDING]) {
```

Comment:
we need this flag to evaluate to `test`, here it's just looking for a truthy value